complaints/forms.py

```diff
@@ -15,17 +18,19 @@ class Meta:
             "avail_start_time",
             "avail_end_time",
         )
-        labels = {
-            "remark": ("Complaint Details"),
-            "contact_no": ("Contact Number"),
-            "room_no": ("Room Number"),
-            "avail_start_time": ("Complaint Details"),
-        }
-        widgets = {"contact_no": forms.Textarea(attrs={"cols": 20, "rows": 1})}
 
 
-class OpenSite_form(forms.ModelForm):
+class HandleForm(forms.ModelForm):
```

Comment:
`Handle` what? Class names should have specific nouns. I'd suggest something like `ComplaintForm`

Reply:
Actually what we plan to do is that the ComplainForm will be used by the user while registering the complaints and the HandleForm will be used by the staff for changing status of the complain or for giving any remark to the user,once the complaint has been resolved.